networking/nw-create-custom-ingress-controller

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+When creating an custom Ingress Controller, 
+
+.Prerequisites
+
+* Must have cluster-admin permissions to the cluster
+* Create a custom wildcard certificate for the custom domain.
+?Do we have a link to this elsewhere...(# oc create secret tls custom-ingress-custom-certs -n openshift-ingress --cert=<certificate.crt> --key=<key.key>)
```

Comment:
https://github.com/openshift/openshift-docs/blob/2291f22e96268bf841f8524c421a99373fc87167/modules/nw-ingress-setting-a-custom-default-certificate.adoc has some content that could be re-used, or possibly refactored into a shared module with the certificate requirements and OpenSSL commands to generate a certificate.